requirements.txt

```diff
@@ -7,6 +7,7 @@ numpy
 netcdf4==1.3
 GDAL~=2.3
 rasterio~=1.0
+sqlalchemy==1.2.18
```

Comment:
Yes, I would definitely support pinning the SA version in general, despite what every problems you are having with v1.3+